flink-runtime/src/main/java/org/apache/flink/runtime/leaderelection/LeaderInformation.java

```diff
@@ -74,7 +78,7 @@ public int hashCode() {
 		return result;
 	}
 
-	public static LeaderInformation known(@Nullable UUID leaderSessionID, @Nullable String leaderAddress) {
+	public static LeaderInformation known(UUID leaderSessionID, String leaderAddress) {
```

Comment:
add assert that `leaderSessionID` and `leaderAddress` is not null. Otherwise this method will allow us to break `isEmpty`.